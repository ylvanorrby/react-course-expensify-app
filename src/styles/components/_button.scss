.button {
    background-color: $blue;
    border: none;
    color: #ffffff;
    display: inline-block;
    font-size: $font-size-large;
    font-weight: 300;
    line-height: 1;
    padding: $s-size;
    text-decoration: none;
}

.button--link {
    background: none;
}

.button--secondary {
    background: #888888;
}