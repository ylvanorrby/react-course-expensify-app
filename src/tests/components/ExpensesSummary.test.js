import React from 'react';
import { shallow } from 'enzyme';
import { ExpensesSummary } from '../../components/ExpensesSummary';

test('Should render summary with 1 expense', () => {
    const wrapper = shallow(
        <ExpensesSummary
            expenseCount={1}
            expensesTotal={123}
        />
    );
    expect(wrapper).toMatchSnapshot();
});

test('Should render summary with multiple expense', () => {
    const wrapper = shallow(
        <ExpensesSummary
            expenseCount={23}
            expensesTotal={2345642}
        />
    );
    expect(wrapper).toMatchSnapshot();
})