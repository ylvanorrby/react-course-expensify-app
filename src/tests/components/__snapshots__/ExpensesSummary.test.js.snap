// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render summary with 1 expense 1`] = `
<div>
  <h1>
    Viewing 
    1
     
    expense
     totalling 
    $1.23
  </h1>
</div>
`;

exports[`Should render summary with multiple expense 1`] = `
<div>
  <h1>
    Viewing 
    23
     
    expenses
     totalling 
    $23,456.42
  </h1>
</div>
`;
